### PR TITLE
ref experimental query editor

### PR DESCRIPTION
**Pull request by scottlepp**
**What this PR does / why we need it**:

In a previous PR, all references to @grafana/experimental were removed and migrated to grafana core as most of those components are no longer experimental and being used throughout core grafana.

The sql editor in experimental is still in active development/fixes.  Rather than having 2 copies, remove from here and add reference to experimental.

Minimize experimental references all over the place by including them in types and exporting from there.

### CODE REVIEW

File: public/app/features/plugins/sql/types.ts
Comment (zoltanbedi) [inline public/app/features/plugins/sql/types.ts:204]: Maybe wait for a release from experimental before we merge this so we can remove this todo.

**Review by zoltanbedi — APPROVED** [review]
LGTM only a thought.

File: public/app/features/plugins/sql/types.ts
Comment (scottlepp) [reply public/app/features/plugins/sql/types.ts]: updated

**Pull request merged**
